source/updateCheck.py

```diff
@@ -98,6 +106,53 @@ def checkForUpdate(auto=False):
 		return None
 	return info
 
+def getPendingUpdate():
+	"""Returns the path to the pending update, if any. Returns C{None} otherwise.
+	@rtype: str
+	"""
+	try:
+		pendingUpdateFile=state["pendingUpdateFile"]
+	except KeyError:
+		return None
+	else:
+		if pendingUpdateFile and os.path.isfile(pendingUpdateFile):
+			return pendingUpdateFile
+		else:
+			state["pendingUpdateFile"] = None
+	return None
+
+def isPendingUpdate():
+	"""Returns whether there is a pending update.
+	@rtype: bool
+	"""
+	return bool(getPendingUpdate())
+
+def executeUpdate(destPath=None):
+	if not destPath:
+		destPath=getPendingUpdate()
+	if not destPath:
+		return
+	state["pendingUpdateFile"]=None
+	state["pendingUpdateVersion"]=None
+	state["removeFile"] = destPath
+	saveState()
+	if config.isInstalledCopy():
+		exeCuteParams = u"--install -m"

```

Comment:
this looks like a bug, here `exeCuteParams` has a capital C, and where it is passed to ShellExecute it does not.